local-cli/bundle/bundle.js

```diff
@@ -8,26 +8,30 @@
  */
 
 const buildBundle = require('./buildBundle');
-const bundleCommandLineArgs = require('./bundleCommandLineArgs');
-const parseCommandLine = require('../util/parseCommandLine');
 const outputBundle = require('./output/bundle');
 const outputPrepack = require('./output/prepack');
+const bundleCommandLineArgs = require('./bundleCommandLineArgs');
 
 /**
  * Builds the bundle starting to look for dependencies at the given entry path.
  */
-function bundleWithOutput(argv, config, output, packagerInstance) {
-  const args = parseCommandLine(bundleCommandLineArgs, argv);
+function bundleWithOutput(argv, config, args, output, packagerInstance) {
   if (!output) {
     output = args.prepack ? outputPrepack : outputBundle;
   }
   return buildBundle(args, config, output, packagerInstance);
-
 }
 
-function bundle(argv, config, packagerInstance) {
```

Comment:
Why command line args are comma separated? Shouldn't be `--entry-file /path/SomeAppBundle.js --platform ios` etc?